packages/edit-site/src/components/page-patterns/use-patterns.js

```diff
@@ -25,38 +23,16 @@ import { store as editSiteStore } from '../../store';
 
 const EMPTY_PATTERN_LIST = [];
 
-const createTemplatePartId = ( theme, slug ) =>
-	theme && slug ? theme + '//' + slug : null;
-
-const templatePartToPattern = ( templatePart ) => ( {
-	blocks: parse( templatePart.content.raw, {
```

Comment:
Reviewed how this (or whether) is being used in the consumer code that uses parts (page-patterns). Everything has been migrated:


- blocks: adapted consumer code
- categories: unused
- description: no need to do anything
- isCustom: unused
- keywords: unused
- id: parts already have an id
- name: removed & adapted the consumer code
- title: removed & adapted the consumer code
- type: as it is
- `_links`: as it is

Reply:
I see actions were also a consumer of this (through dataviews's item). I've only found one case that requires updating (`title` is not using `decodedEntities` https://github.com/WordPress/gutenberg/pull/62927/files#r1658485082 ) but the rest were already ported (`isCustom`, etc.).